src/Halcyon/MemoryRepository.php

```diff
@@ -0,0 +1,172 @@
+<?php namespace October\Rain\Halcyon;
+
+use Illuminate\Cache\Repository;
+
+/**
+ * Provides a simple request-level cache.
+ *
+ * @package october\halcyon
+ * @author Alexey Bobkov, Samuel Georges
+ */
+class MemoryRepository extends Repository
+{
+    /**
+     * Values stored in memory
+     *
+     * @var array
+     */
+    protected $cache = [];
+
+    /**
+     * Retrieve an item from the cache by key.
+     *
+     * @param  string|array $key
+     * @return mixed
```

Comment:
I know I said to remove the `@param $default` that was here earlier but I swear the `$default = null` parameter wasn't present in the method signature when I said that :) Could you re-add it?

Reply:
I think that was in `getFromMemoryCache` I removed it. So it's just missing here :)